builds/e2e/templates/e2e-setup.yaml

```diff
@@ -0,0 +1,140 @@
+steps:
+- checkout: self
+  clean: true
+  fetchDepth: 100
+
+- task: AzureKeyVault@1
```

Comment:
AFAIK when use yaml with azure key vault task to set up VSTS pipeline, it will try to verify your credentail during save to ensure you can access that key vault.  However I have problem before, it seems doesn't detect key vault task inside a template during save; therefore credential is not verified and persisted, and this will cause failure when queuing build.  Do you have this problem?

Reply:
No I didn't have this problem (but I remember when you did)...I guess the fixed it. ¯\_(ツ)_/¯